dwave/cloud/utils.py

```diff
@@ -130,6 +145,25 @@ def uniform_get(sequence, index, default=None):
         return sequence[index] if index < len(sequence) else default
 
 
+def qubo_to_ising(qubo):
```

Comment:
You might want to rename this to distinguish it from say [qubo_to_ising](https://github.com/dwavesystems/dimod/blob/master/dimod/utilities.py#L217) which does the vartype conversion as well.